examples/MySQL/dialect.yml

```diff
@@ -98,21 +158,42 @@ sql functions:
   - name: create timestamp from components
     sql: >
       CASE WHEN $1 > 12 OR $1 < 1 OR $2 > 31 OR $2 < 1 OR
-        $3 > 24 OR $3 < 1 OR $4 > 60 OR $4 < 0 OR $5 > 60 OR $5 < 0 THEN NULL ELSE
+        $3 > 24 OR $3 < 0 OR $4 > 60 OR $4 < 0 OR $5 > 60 OR $5 < 0 THEN NULL ELSE
         STR_TO_DATE(CONCAT($0,'-',$1,'-',$2,' ',$3,':',$4,':',$5),
               CASE WHEN length($0) > 2 THEN '%Y-%m-%d %H:%i:%s' ELSE '%y-%m-%d %H:%i:%s' END)
       END
+# Description: Creates a timestamp from string 2 possible string timestamp format masks 'YYYY-MM-DD hh24:mi:ss' and 'YY-MM-DD hh24:mi:ss'  
```

Comment:
This description doesn't make sense to me. I think it needs separate sentences

Reply:
@vcove Can you suggest the change? Thanks.